cdap-examples/FileSetExample/src/main/java/co/cask/cdap/examples/fileset/FileSetService.java

```diff
@@ -106,6 +109,7 @@ public void read(HttpServiceRequest request, HttpServiceResponder responder,
      */
     @PUT
     @Path("{fileset}")
+    @TransactionPolicy(TransactionControl.EXPLICIT)
```

Comment:
In this PR, the documentation explain explicit transactions. It needs some examples to refer to.